### PR TITLE
FilenameBear.py: Add ``spacecase`` convention

### PR DESCRIPTION
**Pull request by amrandahvur**
Add ``spacecase`` convention for Bear to recognize it

Closes https://github.com/coala/coala-bears/issues/556

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
hey, could you please add the tests in the same commit? simply squash them while rebasing :D

File: tests/general/FilenameBearTest.py [hunk 1]
```diff
@@ -45,6 +45,13 @@ def test_pascal_case(self):
         self.check_validity(self.uut, [""], filename="XYZ/__Init__.py")
         self.check_validity(self.uut, [""], filename="/a/PascalCase")
 
+    def test_space_case(self):
+        self.section["file_naming_convention"] = "space"
+        self.check_validity(self.uut, [""], filename="/Home/xyz/x_y.py",
+                            valid=False)
+        self.check_validity(self.uut, [""], filename="XYZ/__Init__.py")
+        self.check_validity(self.uut, [""], filename="/a/camCase")
```
Comment (amrandahvur) [inline tests/general/FilenameBearTest.py]: Sure, Working on it :D

**Comment by amrandahvur** [comment]
@Adrianzatreanu I have squashed and rebased them, Is the new commit message ok ? Considering the shortlog contains the name of only "FilenameBear.py" and not "FilenameBearTest.py" ??
@Makman2 I have written them, Do I need to add more test cases ??

**Comment by amrandahvur** [comment]
@Nosferatul Done! Please Check ?

**Comment by Nosferatul** [comment]
It's ok, just wait for @Makman2 to see if more tests are required

**Comment by sils** [comment]
ack https://github.com/coala/coala-bears/pull/916/commits/dd82875732853bd2ce4c9c364eb81abddddcc0aa tests are sufficient - the spacecase function tests are more elaborate and shouldn't be duplicated here so it's fine, @Nosferatul why did you mark it WIP? WIP means the author has to change something

**Comment by sils** [comment]
@rultor merge

**Comment by amrandahvur** [comment]
@sils This was my first "difficulty/low" issue. What else do I need to do to become a member ??

**Comment by Nosferatul** [comment]
@sils I didn't know if the tests are sufficient. My bad 😟

**Pull request merged**